### PR TITLE
feat: updates `Breadcrumb`'s glossary entry

### PR DESCRIPTION
**Pull request by kenji-webdev**
### Description

- Adds a `See also` section featuring helpful links related to creating a breadcrumb navigation menu.
- Adds a note suggesting the use of the `<ol>` element.

### Motivation

When I was building a breadcrumb menu, I couldn't find what HTML elements I was supposed to use, both `<p>` and `<ol>` made sense to me, in the beginning I was inclined towards `<p>` as the [HTML5 spec](https://html.spec.whatwg.org/multipage/semantics-other.html#rel-up) recommends it, ultimately I went with `<ol>` as I think it better indicates the hierarchical structure breadcrumbs have.

[Google](https://developers.google.com/search/docs/appearance/structured-data/breadcrumb#json-ld) also uses it in it's HTML example for adding Breadcrumb structured data

### CODE REVIEW

**Review by pepelsbey — CHANGES_REQUESTED** [review]
Hey! Thank you for bringing this up :) It would be great to give some guidance to people looking for a way to mark up their breadcrumbs. Just a few suggestions to make it even better.

**Comment by kenji-webdev** [comment]
Thanks for the suggestions! I'll keep them in mind for future contributions :thumbsup:

Just one more thing, I messed up my last commit message, I wrote `Breadcrumbs` instead of `Breadcrumb`, would it be better for me to `git revert` the commit, force push over it, or should I do something else? Sorry, I'll be more careful next time.

File: files/en-us/glossary/breadcrumb/index.md [hunk 1]
```diff
@@ -13,3 +13,13 @@ A location breadcrumb for this document might look something like this:
 [MDN](/) > [Glossary](/en-US/docs/Glossary) > Breadcrumb
 
 Breadcrumb trails enable users to be aware of their location within a website. This type of navigation, if done correctly, helps users know where they are in a site and how they got there. They can also help a user get back to where they were before and can reduce the number of clicks needed to get to a higher-level page.
+
+> **Note:** While there's no dedicated [semantic HTML element](https://html.spec.whatwg.org/multipage/semantics-other.html#rel-up) for breadcrumb navigation menus, the [`<ol>`](/en-US/docs/Web/HTML/Element/ol) element is commonly used to represent their hierarchical structure.
```
Comment (skyclouds2001) [inline files/en-us/glossary/breadcrumb/index.md:17]:
```suggestion
> **Note:** While there's no dedicated [semantic HTML element](https://html.spec.whatwg.org/multipage/semantics-other.html#rel-up) for breadcrumb navigation menus, the {{htmlelement("ol")}} element is commonly used to represent their hierarchical structure.
```

we could replace the link with macros

**Comment by pepelsbey** [comment]
> Just one more thing, I messed up my last commit message

Don’t bother :) We squash commits when merging, so only the title of the PR will get into the history.

**Review by pepelsbey — APPROVED** [review]
Looks good! Thank you :)

**Comment by pepelsbey** [comment]
Thanks!

**Comment by kenji-webdev** [comment]
Thank you both!

**Pull request merged**